samples/nrf9160/location/Kconfig

```
@@ -34,6 +34,16 @@ config LOCATION_SAMPLE_GNSS_AT_COEX0
 	  Defines what is the AT%XCOEX0 command to be sent to GNSS module. Leave
 	  empty if this command should not be sent.
 
+if NRF_CLOUD_PGPS && !NRF_CLOUD_PGPS_PARTITION
+
+config LOCATION_SAMPLE_PGPS_MCUBOOT_PARTITION
+	bool "Use mcuboot secondary partition for P-GPS data"
```

This isn't supposed to be configured by the user, so maybe we could remove the prompt?
```suggestion
	bool
```